storage/src/archive/mod.rs

```diff
@@ -119,7 +119,8 @@
 //! ```rust
 //! use commonware_runtime::{Spawner, Runner, deterministic::Executor};
 //! use commonware_cryptography::hash;
-//! use commonware_storage::archive::{Archive, Config, translator::FourCap};
+//! use commonware_storage::index::translator::FourCap;
+//! use commonware_storage::archive::{Archive, Config};
```

Comment:
Wonder if we should use a single line for all storage imports?

Reply:
done